geode-dunit/src/main/java/org/apache/geode/management/internal/cli/HeadlessGfsh.java

```diff
@@ -260,16 +259,12 @@ public boolean hasError() {
     public void promptLoop() {
       lock.lock();
       try {
-        while (true) {
-          try {
-            endOfShell.await();
-          } catch (InterruptedException e) {
-            // e.printStackTrace();
-          }
-          this.exitShellRequest = ExitShellRequest.NORMAL_EXIT;
-          setShellStatus(Status.SHUTTING_DOWN);
-          break;
+        try {
+          endOfShell.await();
+        } catch (InterruptedException ignore) {
```

Comment:
Isn't there a new warning for an empty catch block?

Reply:
Not in the compiler but in IJ and other tools. IJ is ok if you name it ignore.